src/Integration.TeamExplorer.UnitTests/Integration.TeamExplorer.UnitTests.csproj

```diff
@@ -5,7 +5,6 @@
 
   <PropertyGroup Label="Specify which external/VS-specific refs are required">
     <RequiresTeamFoundation>true</RequiresTeamFoundation>
-    <RequiresRuleset>true</RequiresRuleset>
```

Comment:
We don't generate or update `.ruleset` files any more, so we don't need to reference those VS assemblies.